app/editor/EditorContentCtrl.js

```diff
@@ -26,6 +23,42 @@
 
     init();
 
+    $rootScope.$on(EventService.EVENT.FILTER_TRANS_UNIT,
+      function (event, filter) {
+        if(filter.status.all === true) {
+          $location.search('status', null);
+        } else {
+          var queryString = '';
+
+          _.forEach(filter.status, function(val, key) {
+            if(val) {
+              queryString += key + ',';
+            }
+          });
+          queryString = queryString.substring(0, queryString.length - 1);
```

Comment:
Oh wait, I just figured it out, it is getting rid of the `','`. I think we should instead put all the keys in an array (e.g. `keys`), then do `queryString = keys.join(',');`.